core/controllers/reader.py

```diff
@@ -329,12 +329,19 @@ def post(self, unused_exploration_id):
         """
         # A domain object representing the old state.
         old_state = exp_domain.State.from_dict(self.payload.get('old_state'))
+        # The id of the exploration to which the state belongs.
+        exp_id = self.payload.get('exp_id')
```

Comment:
You are right, but, if we remove @require_playable and get rid of the unused_exploration_id then this confusing situation would no longer exist.